pve2_api.class.php

```diff
@@ -230,12 +240,12 @@ private function action ($action_path, $http_method, $put_post_parameters = null
 		$action_response_array = json_decode($body_response, true);
 
 		$action_response_export = var_export($action_response_array, true);
-		error_log("----------------------------------------------\n" .
-			"FULL RESPONSE:\n\n{$action_response}\n\nEND FULL RESPONSE\n\n" .
-			"Headers:\n\n{$header_response}\n\nEnd Headers\n\n" .
-			"Data:\n\n{$body_response}\n\nEnd Data\n\n" .
-			"RESPONSE ARRAY:\n\n{$action_response_export}\n\nEND RESPONSE ARRAY\n" .
-			"----------------------------------------------");
+		// error_log("----------------------------------------------\n" .
+		//	"FULL RESPONSE:\n\n{$action_response}\n\nEND FULL RESPONSE\n\n" .
+		//	"Headers:\n\n{$header_response}\n\nEnd Headers\n\n" .
+		//	"Data:\n\n{$body_response}\n\nEnd Data\n\n" .
+		//	"RESPONSE ARRAY:\n\n{$action_response_export}\n\nEND RESPONSE ARRAY\n" .
+		//	"----------------------------------------------");
```

Comment:
Any chance we can put this behind a config of some sort, so we can have the logging during development (for example) without clogging up error logs in production (undoubtedly your goal here)?